Firmware/temperature.cpp

```diff
@@ -153,6 +153,7 @@ uint8_t fanSpeedBckp = 255;
   uint8_t fanState = 0;
 #ifdef EXTRUDER_ALTFAN_DETECT
   bool extruderFanIsAltfan = false; //set to Noctua
+  uint8_t altfanOverride = 0;
```

Comment:
theoretically, we could merge extruderFanIsAltfan and altfanOverride into one var to save 1B of RAM, but at the cost of a slightly more complex code elsewhere.
However, checking if we have an altfan and have an override may actually get shorter:
`if (extruderFanIsAltfan && !altfanOverride) ` into something like `if (altfan == 1)` in case:
#define extruderFanIsAltfanMask 1
#define altfanOverrideMask 2
... if you know what I mean...